app/components/Modal/WithdrawModalNew.jsx

```diff
@@ -551,8 +560,16 @@ class WithdrawModalNew extends React.Component {
         );
     }
 
-    onAssetChanged(value) {
-        value = value.toUpperCase();
+    onAssetChanged(coinItems, selected) {
+        let asset;
```

Comment:
See above.

This code seems to be necessary three times now. Refactoring possible for single point of truth?